### ISSUE TITLE
Bot issue : The bot is blocked on air attack

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/hedgewars#106]
```
What steps will reproduce the problem?
1. Play with a bot
2. ...
3. When the bot choose air attack, it can be blocked sometimes, it takes 40s 
then the bot abandon

v:0.9.14.1 on archlinux x86_64

Attached a video, if this can be useful.


```

-----
Original issue reported on code.google.com by `benpr...@gmail.com` on 20 Nov 2010 at 4:59
* Merged into: #143

Attachments:
* [hedgebug.mkv](https://storage.googleapis.com/google-code-attachments/hedgewars/issue-106/comment-0/hedgebug.mkv)

**Comment by GoogleCodeExporter** [comment]
```
Issue 105 has been merged into this issue.
```

-----
Original comment by `vittorio...@gmail.com` on 20 Nov 2010 at 7:30

**Comment by GoogleCodeExporter** [comment]
-----
Original comment by `sheepyluva` on 2 Feb 2011 at 11:46
* Changed state: **Duplicate**
